src/components/ec-file-dropzone/ec-file-dropzone.spec.js

```diff
@@ -148,5 +156,21 @@ describe('EcFileDropzone', () => {
 
       filesSpy.mockRestore();
     });
+
+    it('@change - should not be emitted when is disabled', async () => {
+      const wrapper = mountFileDropzone({ isDisabled: true });
+
+      await wrapper.trigger('drop', { dataTransfer: { files: filesAndFolders } });
+      expect(wrapper.emitted('change')).toBeUndefined();
+    });
+
+    it('@change - should be emitted  when not disabled', async () => {
```

Comment:
two spaces after emitted and should not we say @change - should be emitted when is not disabled? Thanks!